assets/components/dough_theme/form/_form.scss

```diff
@@ -92,3 +92,27 @@
 .form__list {
   @include body(16, 24);
 }
+
+.form__custom-select {
+  width: 250px;
+  // This is copied from budget planner
+  height: 30px; // custom height
+  border: 1px solid; // $greyscale4; colour is a pacs variable
```

Comment:
should the pacs variable be moved into here as well?

Reply:
Good question. I have left this colour to be defined in PACS/Budget Planner as things stand because I think it should match that context rather than be defined at this level. I think that's the correct approach but I now realise that this colour (initially copied from Budget Planner) should probably be changed in PACS to a colour that already exists in that context. (I'm adding a comment to that effect in the PACS PR). I will delete the comment though to clear up any confusion.

Reply:
Also, the height should be in PACS not here as it is another value specific to PACS not this component.